### ISSUE TITLE
camweb: connecting to SMTP for sending commit e-mails fails often

### ISSUE THREAD

**Issue by mpl** [camlistore/camlistore#980]
The camweb logs show many errors such as:

"Error connecting to SMTP server for sending commit e-mail: dial tcp 50.19.239.94:2500: getsockopt: connection timed out"

These happen in the pollCommits func of website/email.go , when we try to send a commit e-mail message to the Camlistore mailing-list, through a specific SMTP server.

The messages usually eventually succeed, as they evidently show up on the mailing-list, but it sometimes take a while.

More annoyingly, it delays the syncing of the commits from the gerrit repo to the github mirror, since that part happens after we've tried to send all the commit e-mails (so maybe we should decouple these 2 different actions).

The SMTP server we send to is defined as:
*smtpServer = "50.19.239.94:2500" // double firewall: rinetd allow + AWS

**Comment by mpl** [comment]
Alternatively, Brad suggests we could try mailgun:

https://www.mailgun.com/pricing

https://godoc.org/github.com/mailgun/mailgun-go

**Comment by mpl** [comment]
https://camlistore-review.googlesource.com/c/camlistore/+/11946

**Comment by mpl** [comment]
reopening until we now for sure that mailgun can send to mailing lists, which might not be the case.

**Comment by mpl** [comment]
TL;DR is yes we can send to the mailing-lists with mailgun

longer answer is: with the free account we have to add each recipient to the authorized recipients list, which requires said recipient to answer an e-mail, which is not possible for a mailing-list. With the "unity plan", the authorized recipients list is overridden, which makes the above limitation moot.
